lib/omnibus/git_cache.rb

```diff
@@ -196,7 +196,11 @@ def remove_git_dirs
     # @return [Mixlib::Shellout] the underlying command object.
     #
     def git_cmd(command)
-      shellout!("git -c core.autocrlf=false --git-dir=#{cache_path} --work-tree=#{install_dir} #{command}")
+      shellout!("git -c core.autocrlf=false \
+                     -c core.ignorecase=false \
+                     --git-dir=#{cache_path} \
+                     --work-tree=#{install_dir} \
+                     #{command}".gsub(/\s+/, " "))
```

Comment:
If `install_dir` contains spaces/isn't already escaped properly, within the git_cache is probably not the correct place to be sanitizing it. https://github.com/chef/omnibus/blob/master/lib/omnibus/util.rb#L88-L92 would be a better spot.